<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>767</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="videoGraphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>9</y>
      <width>511</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxRecognition">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>220</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>识别控制</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>232</width>
       <height>119</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="btnLoadTestData">
        <property name="text">
         <string>加载测试数据</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnTurnOnRecognition">
        <property name="text">
         <string>开识别Crtl+U</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+U</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnUnloadTestData">
        <property name="text">
         <string>卸载测试数据</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnTurnOffRecognition">
        <property name="text">
         <string>关识别Crtl+I</string>
        </property>
        <property name="shortcut">
         <string>Ctrl+I</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="btnStartAutoTrack">
        <property name="text">
         <string>自动跟踪</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnStopAutoTrack">
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxWelding">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>231</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>焊接过程信息</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>82</width>
       <height>89</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="labelSeamWidth">
        <property name="text">
         <string>焊缝宽度：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelSeamRemainHeight">
        <property name="text">
         <string>焊缝余高：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelOffSet">
        <property name="text">
         <string>焊缝偏差：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>31</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="unitLabel0">
        <property name="text">
         <string>mm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="unitLabel1">
        <property name="text">
         <string>mm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="unitLabel2">
        <property name="text">
         <string>mm</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>152</width>
       <height>58</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelCameraState">
        <property name="text">
         <string>相机状态：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_CameraState">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;未连接&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelCameraResolution">
        <property name="text">
         <string>相机分辨率：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_CameraResolution">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt;未知&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>71</width>
       <height>89</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_WeldSeamWidth">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#5500ff;&quot;&gt;？&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_WeldSeamRemainHeight">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#5500ff;&quot;&gt;？&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_WeldSeamOffset">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#5500ff;&quot;&gt;？&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>767</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>参数</string>
    </property>
    <addaction name="actionParameterSet"/>
    <addaction name="actionWeldParameter"/>
    <addaction name="actionPID"/>
   </widget>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>控制</string>
    </property>
    <addaction name="actionManualControl"/>
   </widget>
   <widget class="QMenu" name="menuTrack">
    <property name="title">
     <string>跟踪</string>
    </property>
    <addaction name="actionOfflineTrack"/>
   </widget>
   <widget class="QMenu" name="menuCalibration">
    <property name="title">
     <string>校正</string>
    </property>
    <addaction name="actionCalibration1"/>
   </widget>
   <addaction name="menuCamera"/>
   <addaction name="menuControl"/>
   <addaction name="menuTrack"/>
   <addaction name="menuCalibration"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionParameterSet">
   <property name="text">
    <string>相机</string>
   </property>
  </action>
  <action name="actionManualControl">
   <property name="text">
    <string>手动控制</string>
   </property>
  </action>
  <action name="actionOfflineTrack">
   <property name="text">
    <string>离线跟踪</string>
   </property>
  </action>
  <action name="actionWeldParameter">
   <property name="text">
    <string>焊接</string>
   </property>
  </action>
  <action name="actionCalibration1">
   <property name="text">
    <string>简单校正</string>
   </property>
  </action>
  <action name="actionPID">
   <property name="text">
    <string>PID</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
